Stateczki/Game.cs

```diff
@@ -6,7 +6,16 @@ class Game
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Square[,] exampleOcean = new Square[3,3];
+            for (int i = 0; i < 3; i++)
```

Comment:
Foreach would be easier / more readable?

Reply:
Also, I think we should first merge PR # 2, then we can reference Ocean from here (instead of new Square[,]). Or we can refactor this after we merge both branches.

Reply:
One last thing - what happens, when you initialize new Square[,]? Doesn't it automatically fill the array with new Squares? Not sure how that works.